main.go

```diff
@@ -31,6 +35,7 @@ const (
 	useCacheArg            = "use-cache"
 	applyToGeneratedFiles  = "apply-to-generated-files"
 	excludesArg            = "excludes"
+	modulePathRegex        = `^github.com/\w+/goimports-reviser(/v\d+)?@?`
```

Comment:
About the w+, I have a few doubt.

I assume you want to allow anyone to fork your repository. Which is great, but `\w` is `[A-Za-z0-9_]`, so `-` is absent

I would use


```suggestion
	modulePathRegex        = `^github.com/[^/]+/goimports-reviser(/v\d+)?@?`
```

Reply:
Usernames for github accounts [may only contain alphanumeric character and dashes](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-iam/iam-configuration-reference/username-considerations-for-external-authentication#about-username-normalization) -- I really do think `\w+` is appropriate here.
<img width="531" alt="image" src="https://github.com/user-attachments/assets/6eaa49fc-74ba-465d-bb9e-ea295b20c9e3">

Reply:
This is unlikely to ever change, either, because github usernames get used as hostnames for github's hosted content features, e.g. I could set up "n-oden.github.io" but `n_oden.github.io` is not a valid hostname.

Reply:
Thanks for checking, but as I said \w doesn't contains `-`.

So any, user with a - won't be able to use the version feature

Reply:
d'oh, you are of course quite correct.  updated.